src/wsock_message.erl

```diff
@@ -357,6 +357,8 @@ process_frame(open_close, _, Frame, Message) ->
   frame_to_complete_message(Frame, Message);
 process_frame(open_continue, _, Frame, Message) ->
   frame_for_fragmented_message(Frame, Message);
+process_frame(continue, begin_message, Frame, Message) ->
```

Comment:
@ctradu hi. I know it has been a long time since you opened this pull request but now that I'm back I would like to know the reasoning behind this change.

How could you have a Frame that's supposed to `continue` (be part of an ongoing message) and at the same time consider it as part of a new message.